modules/expo-localization/android/build.gradle

```diff
@@ -0,0 +1,73 @@
+
+buildscript {
+    repositories {
+        jcenter()
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.3'
+    }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'maven'
+
+group = 'host.exp.exponent'
+version = '1.0.2'
```

Comment:
Let's start with `1.0.0` 🙂

Reply:
I just ran `cd expo/modules; node generate_new_module.js`. Didn't set the version manually